packages/core/test/Form_test.js

```diff
@@ -2966,9 +3013,13 @@ describe("Form omitExtraData and liveOmit", () => {
       properties: {
         foo: { type: "string" },
         bar: { type: "string" },
+        add: {
```

Comment:
Can you actually keep this test the same, then make another test that specifically tests "should not omit additionalProperties on change with omitExtraData=true and liveOmit=true"?